From 007d23e345fb8c8f5952216aa1f44c96b8808398 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?K=C3=A9vin=20L=27h=C3=B4pital?=
 <kevin.lhopital@savoirfairelinux.com>
Date: Thu, 31 Mar 2022 13:12:00 +0200
Subject: [PATCH 1/4] media/decoder_finder.h: use h264_imxvpuapi to decode h.264

Always use the ffmpeg hardware decoder h264_imxvpuapi when decoding
h.264.
---
 src/media/decoder_finder.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/media/decoder_finder.h b/src/media/decoder_finder.h
index 55b3b4d..ac26e33 100644
--- a/src/media/decoder_finder.h
+++ b/src/media/decoder_finder.h
@@ -46,6 +46,8 @@ findDecoder(const enum AVCodecID codec_id)
     case AV_CODEC_ID_VP9:
         codec_name = "vp9_mediacodec"; break;
 #endif
+    case AV_CODEC_ID_H264:
+        codec_name = "h264_imxvpuapi"; break;
     default:
         codec_name = nullptr;
     }
-- 
2.25.1

